Languages/Japanese/DefInjected/MoreFactionInteraction.EventDef/AnnualExpoEvents.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+  <MFI_GameOfUrComp.label>ウルのゲーム大会</MFI_GameOfUrComp.label>
+  <MFI_ShootingComp.label>射撃競技会</MFI_ShootingComp.label>
+  <MFI_CulturalSwap.label>文化交流</MFI_CulturalSwap.label>
+  <MFI_ScienceFaire.label>科学博覧会</MFI_ScienceFaire.label>
+  <MFI_AcousticShow.label>野外コンサート</MFI_AcousticShow.label>
+

```

Comment:
My apologies! I didn't have the right structure set up for this and didn't realise it until I saw your Pull Request. 

Can you add to this file

        <MFI_GameOfUrComp.outComeFirstPlace>
        <MFI_GameOfUrComp.outcomeFirstLoser>
        <MFI_GameOfUrComp.outComeFirstOther>

etc - they are (roughly) the same as the one from Languages/Japanese/Keyed/Keys.xml

Also please add:

        <MFI_GameOfUrComp.theme>the Royal Game-of-Ur</MFI_GameOfUrComp.theme>
        <MFI_GameOfUrComp.themeDesc>Send your best gamer.</MFI_GameOfUrComp.themeDesc>
for each event.

Thank you!